dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -101,23 +104,31 @@ protected BurstDateFormat initialValue() {
         }
     };
 
-    protected static boolean addToJSON(StringBuilder sb, String name, String value, boolean jsonEscapeName,
+    protected static boolean addToJSON(StringBuilder sb, String name, String value, String eventType, boolean jsonEscapeName,
                                        boolean jsonEscapeValue, boolean trim, boolean isFirstField) {
 
-        boolean b = addToJSON(sb, name, value, jsonEscapeName, jsonEscapeValue, trim, isFirstField, false);
+        boolean b = addToJSON(sb, name, value, eventType, jsonEscapeName, jsonEscapeValue, trim, isFirstField, false);
         return b;
     }
 
-    protected static boolean addToJSON(StringBuilder sb, String name, String value, boolean jsonEscapeName,
+    protected static boolean addToJSON(StringBuilder sb, String name, String value, String eventType, boolean jsonEscapeName,
                                        boolean jsonEscapeValue, boolean trim, boolean isFirstField, boolean isQuoteless) {
 
+        Map<String, Set<String>> omitFieldsMap = BaseTraceService.getOmitFieldsMap();
```

Comment:
Instead of calling BaseTraceService Map, might be better to store omitFieldsMap into the classes (LogTraceData, accessLogData) as an Arraylist.